### ISSUE TITLE
Web-Frontend not accessible any more in community edition

### ISSUE THREAD

**Issue by closertotheedge** [ntop/ntopng#4903]
ntopng web-frontend not accessible after 10 minutes when going from enterprise to community edition. NO problems during first 10 minutes!

Google Chrome says "ERR_TOO_MANY_REDIRECTS"

tail -f /var/log/ntopng.log gives a lot of messages like:

Jan 13 10:11:37 rpi-ntopng ntopng: [LuaEngine.cpp:1159] WARNING: Script failure [/usr/share/ntopng/scripts/lua/http_status_code.lua][error loading module 'alert_iec_invalid_transition' from file '/var/lib/ntopng/plugins1/alert_definitions/alert_iec_invalid_transition.lua':#012#011...gins1/alert_definitions/alert_iec_invalid_transition.lua:1: unexpected symbol near '<\135>']

ntong running on raspi4 2gb, invoked by fritzdump.sh. 

ntopng package is ntopng_4.3.210111-12741_armhf.deb. Same problem when downgrading to ntopng_4.3.210110-12724_armhf.deb.

**Comment by simonemainardi** [comment]
try and add option `--community` if you want to run it in community version

**Comment by closertotheedge** [comment]
I added --community to fritzdump.sh and the error messages are gone. So the problem is fixed for now.
Nevertheless I think it is a strange behaviour because old ntopng-versions worked flawlessly without this argument for months, going automatically and without errors after 10 minutes from enterprise to community edition...

**Comment by rreiner** [comment]
I have this problem too.  It worked perfectly without the --community flag on all previous versions, but the web GUI now becomes inaccessible after 10 minutes.

**Comment by simonemainardi** [comment]
I am unable to reproduce reported behavior. @rreiner are you also on armhf? Please show me the output of ntopng --version

**Comment by rreiner** [comment]
~$ ntopng --version
v.4.3.210117    [Enterprise/Professional/Embedded build]
GIT rev:        dev:406f933f207b53f74c525c4a41f87a30087d80c8:20210117
Pro rev:        r3506
Built on:       Raspbian GNU/Linux 10 (buster)
System Id:      FD51A59B499602D2
Platform:       armv7l
Edition:        Enterprise Embedded
License Type:   Time-Limited [Empty license file]
Validity:       Until Tue Jan 19 12:23:59 2021

**Comment by rreiner** [comment]
The URL http://pinkeye:3000/ works fine for the first ten minutes.
When accessing the web GUI after the initial 10 minutes, it gets into a redirect loop.

The final URL in the browser after it gives up trying to follow the redirects is:

http://pinkeye:3000/lua/http_status_code.lua?message=internal_error&referer=pinkeye%3A3000%2Flua%2Flogin.lua%3Freferer%3Dpinkeye%253A3000%252Flua%252Fhttp_status_code.lua%253Fmessage%253Dinternal_error%2526referer%253Dpinkeye%25253A3000%25252Flua%25252Flogin.lua%25253Freferer%25253Dpinkeye%2525253A3000%2525252Flua%2525252Fhttp_status_code.lua%2525253Fmessage%2525253Dinter&error_message=error%20loading%20module%20%27alert_iec_invalid_transition%27%20from%20file%20%27%2Fmnt%2Fntopngdata%2Fplugins1%2Falert_definitions%2Falert_iec_invalid_transition.lua%27%3A%0A%09...gins1%2Falert_definitions%2Falert_iec_invalid_transition.lua%3A1%3A%20unexpected%20symbol%20near%20%27%3C%5C135%3E%27

**Comment by rreiner** [comment]
The above is the final URL in firefox.  It's different in chrome:

http://pinkeye:3000/lua/login.lua?referer=pinkeye%3A3000%2Flua%2Fhttp_status_code.lua%3Fmessage%3Dinternal_error%26referer%3Dpinkeye%253A3000%252Flua%252Flogin.lua%253Freferer%253Dpinkeye%25253A3000%25252Flua%25252Fhttp_status_code.lua%25253Fmessage%25253Dinternal_error%252526referer%25253Dpinkeye%2525253A3000%2525252Flua%2525252Flogin.lu

**Comment by simonemainardi** [comment]
Thanks for providing this. Are you using it with fritzdump as well?

**Comment by rreiner** [comment]
No, I'm not using fritzdump

**Comment by mihakralj** [comment]
have the same issue. The --community flag works - for about an hour or so. But even with --community flag it goes into redirect loop after about an hour or so and then ntopng service needs to be restarted to make GUI operational again.

```
admin@OPNsense:~ % ntopng --version
v.4.3.210120    [Enterprise/Professional build]
GIT rev:        dev:80ed58464e3570d144620a502e69f8723b8bceab:20210120
Pro rev:        r3509
Built on:       FreeBSD 12.2
System Id:      7B598744920BAB21
Platform:       amd64
Edition:        Enterprise L
License Type:   Time-Limited [Empty license file]
Validity:       Until Wed Jan 20 15:30:01 2021
```

**Comment by simonemainardi** [comment]
@mihakralj and you get the same errors such as 

> unexpected symbol near '<\135>'

?

**Comment by mihakralj** [comment]
this is the final redirected URL after ntopng flips from Enterprise to community edition:

https://192.168.1.1:3443/lua/http_status_code.lua?message=internal_error&referer=192.168.1.1%3A3443%2Flua%2Fhttp_status_code.lua%3Fmessage%3Dinternal_error%26referer%3D192.168.1.1%253A3443%252Flua%252Fhttp_status_code.lua%253Fmessage%253Dinternal_error%2526referer%253D192.168.1.1%25253A3443%25252Flua%25252Fhttp_status_code.lua%25253Fmessage%25253Dinternal_error%252526refer&error_message=error%20loading%20module%20%27alert_iec_invalid_transition%27%20from%20file%20%27%2Fvar%2Fdb%2Fntopng%2Fplugins1%2Falert_definitions%2Falert_iec_invalid_transition.lua%27%3A%0A%09...gins1%2Falert_definitions%2Falert_iec_invalid_transition.lua%3A1%3A%20unexpected%20symbol%20near%20%27%3C%5C135%3E%27

**Comment by mihakralj** [comment]
here is the bottom of ntopng.log at the time of failure:
`
21/Jan/2021 14:05:10 [LuaEngine.cpp:391] WARNING: Script failure [/var/db/ntopng/plugins1/alert_definitions/alert_iec_invalid_transition.lua][...gins1/alert_definitions/alert_iec_invalid_transition.lua:1: unexpected symbol near '<\135>']
21/Jan/2021 14:05:10 [AlertCheckLuaEngine.cpp:73] WARNING: Script failure[/usr/local/share/ntopng/scripts/callbacks/interface/network.lua] [.../local/share/ntopng/scripts/lua/modules/alert_consts.lua:353: module 'alert_iec_invalid_transition' not found:]
21/Jan/2021 14:05:10 [LuaEngine.cpp:391] WARNING: Script failure [/var/db/ntopng/plugins1/alert_definitions/alert_iec_invalid_transition.lua][...gins1/alert_definitions/alert_iec_invalid_transition.lua:1: unexpected symbol near '<\135>']
21/Jan/2021 14:05:10 [AlertCheckLuaEngine.cpp:73] WARNING: Script failure[/usr/local/share/ntopng/scripts/callbacks/interface/host.lua] [.../local/share/ntopng/scripts/lua/modules/alert_consts.lua:353: module 'alert_iec_invalid_transition' not found:]
21/Jan/2021 14:05:10 [AlertCheckLuaEngine.cpp:73] WARNING: Script failure[/usr/local/share/ntopng/scripts/callbacks/interface/host.lua] [error loading module 'alert_iec_invalid_transition' from file '/var/db/ntopng/plugins1/alert_definitions/alert_iec_invalid_transition.lua':
        ...gins1/alert_definitions/alert_iec_invalid_transition.lua:1: unexpected symbol near '<\135>']
21/Jan/2021 14:05:10 [AlertCheckLuaEngine.cpp:73] WARNING: Script failure[/usr/local/share/ntopng/scripts/callbacks/interface/host.lua] [error loading module 'alert_iec_invalid_transition' from file '/var/db/ntopng/plugins1/alert_definitions/alert_iec_invalid_transition.lua':
        ...gins1/alert_definitions/alert_iec_invalid_transition.lua:1: unexpected symbol near '<\135>']
21/Jan/2021 14:05:14 [LuaEngine.cpp:1159] WARNING: Script failure [/usr/local/share/ntopng/scripts/lua/rest/v1/get/interface/data.lua][error loading module 'alert_iec_invalid_transition' from file '/var/db/ntopng/plugins1/alert_definitions/alert_iec_invalid_transition.lua':
        ...gins1/alert_definitions/alert_iec_invalid_transition.lua:1: unexpected symbol near '<\135>']
`

**Comment by simonemainardi** [comment]
Issue should be resolved. New arm builds are in progress and will be ready in a couple of hours from now. Please, update, and let me know. Thanks for your support in troubleshooting this.

**Comment by mihakralj** [comment]
just FYI, my ntopng runs on freeBSD on x86 Intel architecture...

**Comment by closertotheedge** [comment]
Thanks for the new version. I can confirm that the problem is solved with Version 4.3.210125-12909 / "ntopng / Embedded v.4.3.210125 (Raspbian GNU/Linux 10 (buster))".

What I also noticed is that the German web-frontend has some translation errors. For example "Upgrade to Pro/Enterprise version" in English now gives "Upgralang.zu lang. Professional Version" in German which is complete nonsense.

**Comment by simonemainardi** [comment]
@closertotheedge thanks for reporting. 

As per the german translation, you may want to check the lang file out, fix it, and send a pull request. That would be great.

This is the file: https://github.com/ntop/ntopng/blob/dev/scripts/locales/de.lua

Thank you.
